android/capacitor/src/main/java/com/getcapacitor/BridgeWebViewClient.java

```diff
@@ -56,6 +56,15 @@ public void onReceivedError(WebView view, WebResourceRequest request, WebResourc
                 listener.onReceivedError(view);
             }
         }
+
+        String errorPath = bridge.getErrorUrl();
+        String appUrl = bridge.getAppUrl();
+        if (appUrl.charAt(appUrl.length() - 1) != '/') {
+            appUrl += "/";
+        }
+        if (errorPath != null && request.getUrl().toString().equals(appUrl)) {
```

Comment:
Why does this compare if the url to load is different from the appUrl? on iOS it's not comparing that.
If it's because when loading the error page the app enters into a loop of failures because of favicon.ico not loading, I think it's best to check if the request was to the main frame
`if (errorPath != null && request.isForMainFrame()) {`.

Reply:
> If it's because when loading the error page the app enters into a loop of failures..
Yes, this was exactly why.  Thanks!